templates/travis/.travis/install.sh.j2

```diff
@@ -69,6 +76,8 @@ images:
       plugins:
         - pulp-certguard
```

Comment:
Let's remove pulp-certguard and have it be provided by the additional_plugins config for each plugin.